src/components/Icons/iconChangelogData.ts

```diff
@@ -24,8 +31,8 @@ export const changelogData: ChangelogData[] = [
     notes: ["Added the navigationMoreVert (vertical ellipsis) icon."],
   },
   {
-    date: "Prerelease",
-    version: "Prerelease",
+    date: "2024-01-24",
+    version: "2.1.4",
```

Comment:
Ugh I thought I was meticulous about updating all files but I guessed I missed this.